UNRELEASED.md

```diff
@@ -2,6 +2,10 @@
 
 > Place your changes below this line.
 
+**Fixed:**
```

Comment:
Is this breaking? Consumers who were using `float` in CSS before will need to change it, right? 
```suggestion
**Breaking:**
```

Reply:
I don't think so — the button was absolutely positioned before, so `float` would have had no effect. Here's proof (this was recorded on `master`)

![Kapture 2020-11-17 at 17 27 04](https://user-images.githubusercontent.com/73652/99425008-3e793d00-28fa-11eb-9ff4-3ae17552f24a.gif)